src/frontend/handlers.go

```diff
@@ -79,8 +80,20 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		ps[i] = productView{p, price}
 	}
 
-	//get env and render correct platform banner.
-	var env = os.Getenv("ENV_PLATFORM")
+	// Set ENV_PLATFORM (checks for GCP. otherwise uses env value.)
+	var env = "local"
+	addrs, err := net.LookupHost("metadata.google.internal.")
+	if err == nil && len(addrs) >= 0 {
+		log.Debugf("Detected Google metadata servers: %v, setting ENV_PLATFORM to GCP.", addrs)
+		env = "gcp"
+	} else {
+		var envFromConfig = os.Getenv("ENV_PLATFORM")
+		fmt.Printf("⭐️ Env from config is: %s", envFromConfig)
```

Comment:
Do we have other emoji's in print statements? I love emoji! Just making sure we stay consistent. 😄

Reply:
nope that was just for debugging, good catch! removing.